includes/learnopengl/model.h

```diff
@@ -26,6 +26,8 @@ unsigned int TextureFromFile(const char *path, const string &directory, bool gam
 class Model 
 {
 public:
+    static unsigned int defaultDiffuse, defaultSpecular, defaultNormal, defaultOther;
```

Comment:
This can function in the private section. Unless you have external texture you want to use.
There's a way for that too!

Reply:
The issue with this is the `textureFromFile()` function is outside of the model class so when it tries to return `Model::defaultDiffuse` it can't because that is a private member.